tests/test-cases/Group23-VIC-Machine-Service/23-04-VCH-Inspect.robot

```diff
@@ -0,0 +1,145 @@
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#	http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+*** Settings ***
+Documentation     Test 23-04 - VCH Inspect
+Resource          ../../resources/Util.robot
+Resource          ../../resources/Group23-VIC-Machine-Service-Util.robot
+Suite Setup       Setup
+Suite Teardown    Teardown
+
+Default Tags
+
+*** Keywords ***
+Setup
+    Start VIC Machine Server
+    Install VIC Appliance To Test Server  certs=${true}  additional-args=--ops-user=%{TEST_USERNAME} --ops-password=%{TEST_PASSWORD} --debug 1
+
+    ${id}=  Get VCH ID  %{VCH-NAME}
+    ${dc-id}=  Get Datacenter ID
+
+    Set Suite Variable  ${VCH-ID}  ${id}
+    Set Suite Variable  ${DC-ID}  ${dc-id}
+
+Teardown
+    Cleanup VIC Appliance On Test Server
+    Terminate All Processes    kill=True
+
+Inspect VCH
+    Get Path Under Target  vch/${VCH-ID}
+
+Inspect VCH Using Session
+    Get Path Under Target Using Session  vch/${VCH-ID}
+
+Inspect VCH Within Datacenter
+    Get Path Under Target    datacenter/${DC-ID}/vch/${VCH-ID}
+
+Inspect VCH Within Datacenter Using Session
+    Get Path Under Target Using Session    datacenter/${DC-ID}/vch/${VCH-ID}
+
+Verify VCH Inspect Output
+    # debug
+    Property Should Be Equal        .debug  1
+
+    # networks
+    Property Should Be Equal        .network.bridge.ip_range                    172.16.0.0/12
+    Property Should Not Be Empty    .network.bridge.port_group
+    Property Should Be Equal        .network.public.nameservers[0]              null
+    Property Should Not Be Empty    .network.public.port_group
+    Property Should Be Equal        .network.container[0].alias                 public
+    Property Should Not Be Empty    .network.container[0].port_group
+
+    # cert
+    ${domain}=  Get Environment Variable  DOMAIN  ''
+    Run Keyword If  $domain != ''   Property Should Contain    .auth.server.certificate.pem  -----BEGIN CERTIFICATE-----
+    Property Should Be Equal        .auth.server.private_key.pem                null
+
+    # compute
+    Property Should Not Be Empty    .compute.resource.id
+
+    # storage
+    Property Should Be Equal        .storage.base_image_size.value              8000000
+    Property Should Be Equal        .storage.base_image_size.units              KB
+
+    Property Should Contain         .storage.image_stores[0]                    %{TEST_DATASTORE}
+    Property Should Contain         .storage.volume_stores[0].datastore         %{TEST_DATASTORE}/%{VCH-NAME}-VOL
+    Property Should Be Equal        .storage.volume_stores[0].label             default
```

Comment:
Should we also be verifying that `.image_stores` and `.volume_stores` contain only a single element? (We verify that the first element of each is what we're expecting, but the test would still pass if there were other bogus values showing up in the response.)

Solving this elegantly may require adding a new `Property Should ...` keyword (perhaps along the lines of `Property Should Be Unary`, which we could use for each array we expect to have a single element, or perhaps along the lines of `Property Length Should Be`, which we could use for each array we expect to have a specific length).